Library/Formula/zeromq.rb

```diff
@@ -2,7 +2,7 @@ class Zeromq < Formula
   homepage "http://www.zeromq.org/"
   url "http://download.zeromq.org/zeromq-4.0.5.tar.gz"
```

Comment:
You can kill the ` url ` and ` sha1 ` lines here now they're nested inside the ` stable do ` block :).